### ISSUE TITLE
layman or overlays might be a useful submodule. 

### ISSUE THREAD

**Issue by necrose99** [nrdvana/squash-portage#1]
/var/lib/layman.sqsh after a few overlays including if you fork a repo , 20 MB hear or thier 

with ZFS I can make it more transparent however laptops etc, better to use sqashfs.

**Comment by nrdvana** [comment]
I've never used layman, so can't comment... do you have a suggestion for how this tool could be enhanced to work with layman?

**Comment by necrose99** [comment]
**namely just to do the same to /var/lib/layman/\* *\* 
layman -L , Layman -a repo name , Layman -S 
make.conf + repo names xml or repo's installed , and repo list xml snapshot. 
make.conf is then referenced as overlay in primary make.conf. 
but basicly in /var/lib/layman/ is a set of mini portage dirs....  also for backing up your layman as squashed snapshots may also prove useful.  or the ability to clone easily to other boxes etc.  
**[http://gpo.zugaina.org/Overlays](http://gpo.zugaina.org/Overlays)**
https://github.com/alexdux/async-emerge-overlay , uses simular techs for in ram overlay fs /aufs 3/4 for spwning a chroot for packages 
also builds upgrades , esp good if system is old and too much use flags have changed, can do a full world in flying chroot , which deposits packages in /packages dir and the chroot disappears. 
might have a few things you could filch.  ie mounting the squashfile elsewhare , and then mount read right overlayfs aand sync to squash as needed. 
http://unix.stackexchange.com/questions/80305/mounting-a-squashfs-filesystem-in-read-write
http://unix.stackexchange.com/questions/252109/mount-overlayfs-as-read-write-fs
https://www.brunsware.de/blog/gentoo/portage-tree-squashfs-overlayfs.html 
not necessary in boot terms .. but you could overly the fs and dynamicaly sync changes to squash-file/s 
anyhow if your a sabayon user (Binary Gentoo) you may not use as much, Pentoo often has security packages, however their are numbers more...  less your a heavy a builder... then they catch dust. 

as you have going on for portage, theirs a number of repos in layman including a few of mine. 
sabayon , pentoo , funtoo , etc have variant packages or testing package necromancy**\* (i need time to de-cruft and or fix)  i have  few.... 
but add about 20 good repos ..... as a security Engineer , I have a slight...make that a  major addiction to testing any packages... related to Cypto end users can use and any toys that make audits, pen-testing, forensics, Security Admin work Easier... 
**bad news is laptop hard-dive space..**. likewise can chew a few megs to 20 megs a repo x 20 or so **can easily chew up a few gigs**. as a heavy dev... or tinkerer , and if only doing on week ends  or less often its allot of disk-space being used up. and if using docker , also not wanting to use up space on a cloud box. 

other than /usr/portage/packages  could likely move them to /var/portage/packages and drop in a symlink anyhow or /home/portage/packages or something and symlink , as trying to compress them wont really matter  much.
